project/Http4sPlugin.scala

```diff
@@ -249,54 +249,59 @@ object Http4sPlugin extends AutoPlugin {
+  lazy val discipline                       = "org.typelevel"          %% "discipline"                  % "0.9.0"
+  lazy val fs2Io                            = "co.fs2"                 %% "fs2-io"                      % "0.10.4"
+  lazy val fs2ReactiveStreams               = "com.github.zainab-ali"  %% "fs2-reactive-streams"        % "0.5.1"
+  lazy val fs2Scodec                        = "co.fs2"                 %% "fs2-scodec"                  % fs2Io.revision
+  lazy val gatlingTest                      = "io.gatling"             %  "gatling-test-framework"      % "2.3.1"
+  lazy val gatlingHighCharts                = "io.gatling.highcharts"  %  "gatling-charts-highcharts"   % gatlingTest.revision
+  lazy val http4sWebsocket                  = "org.http4s"             %% "http4s-websocket"            % "0.2.1"
+  lazy val javaxServletApi                  = "javax.servlet"          %  "javax.servlet-api"           % "3.1.0"
+  lazy val jawnJson4s                       = "org.spire-math"         %% "jawn-json4s"                 % "0.11.1"
+  lazy val jawnFs2                          = "org.http4s"             %% "jawn-fs2"                    % "0.12.2"
+  lazy val jettyServer                      = "org.eclipse.jetty"      %  "jetty-server"                % "9.4.9.v20180320"
+  lazy val jettyServlet                     = "org.eclipse.jetty"      %  "jetty-servlet"               % jettyServer.revision
+  lazy val json4sCore                       = "org.json4s"             %% "json4s-core"                 % "3.5.3"
+  lazy val json4sJackson                    = "org.json4s"             %% "json4s-jackson"              % json4sCore.revision
+  lazy val json4sNative                     = "org.json4s"             %% "json4s-native"               % json4sCore.revision
+  lazy val jspApi                           = "javax.servlet.jsp"      %  "javax.servlet.jsp-api"       % "2.3.1" // YourKit hack
+  lazy val log4s                            = "org.log4s"              %% "log4s"                       % "1.6.1"
+  lazy val logbackClassic                   = "ch.qos.logback"         %  "logback-classic"             % "1.2.3"
+  lazy val metricsCore                      = "io.dropwizard.metrics"  %  "metrics-core"                % "4.0.2"
+  lazy val metricsJson                      = "io.dropwizard.metrics"  %  "metrics-json"                % metricsCore.revision
+  lazy val nettyHandler                     = "io.netty"               % "netty-handler"                % "4.1.24.Final"
+  lazy val nettyHttpCodec                   = "io.netty"               % "netty-codec-http"             % nettyHandler.revision
+  lazy val nettyNativeTransport             = "io.netty"               % "netty-transport-native-epoll" % nettyHandler.revision classifier "linux-x86_64"
+  lazy val nettyUnixCommon                   = "io.netty"              % "netty-transport-native-unix-common" % nettyHandler.revision
```

Comment:
This is why I hate vertical alignment.

If we're blowing out the diff for netty-transport-native-epoll, might as well go all the way for this one, no?